cmd/kyma/alpha/sign/sign.go

```diff
@@ -0,0 +1,20 @@
+package sign
+
+import (
+	"github.com/kyma-project/cli/cmd/kyma/alpha/sign/module"
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/spf13/cobra"
+)
+
+//NewCmd creates a new Kyma CLI command
+func NewCmd(o *cli.Options) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "sign",
+		Short: "Sign all module resources from unsigned component descriptor which hosted in a remote OCI registry",
```

Comment:
```suggestion
		Short: "Signs all module resources from an unsigned component descriptor that's hosted in a remote OCI registry",
```

What's the difference to the short description of sign/module/module.go ? They seem identical to me.

Reply:
They are displayed in different level of help message, `Short` one will show when you call `kyma-cli --help` in the list of available commands , `Long` will display when you call the specific command, in this case `kyma-cli sign --help`

Reply:
The question is why is the `Short` description identical in both files.
My question is **_not_** why there is a `Short` and a `Long` description for each command.

Reply:
ok, the short in `sign/module/module.go` is displayed in document, but not in cli command, check here: https://github.com/kyma-project/cli/pull/1319/files#diff-2b803938f5f74eaed480147d6cf6134a002dcb323a107e26860cee70e337e888R5